theseus/src/state/profiles.rs

```diff
@@ -97,6 +105,7 @@ impl Profile {
         }
 
         Ok(Self {
+            uuid: create_uuid(),
```

Comment:
This should take a UUID as a parameter as ideally the profile directory name and the UUID should always match up